bridge/pkg/terra/terra.go/client/lcdclient.go

```diff
@@ -0,0 +1,75 @@
+package client
+
+import (
+	"github.com/certusone/wormhole/bridge/pkg/terra/terra.go/key"
+	"github.com/certusone/wormhole/bridge/pkg/terra/terra.go/msg"
+	"github.com/certusone/wormhole/bridge/pkg/terra/terra.go/tx"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// LCDClient outer interface for building & signing & broadcasting tx
+type LCDClient struct {
+	URL           string
+	ChainID       string
+	GasPrice      msg.DecCoin
+	GasAdjustment msg.Dec
+
+	TmKey key.StdPrivKey
+}
+
+// NewLCDClient create new LCDClient
+func NewLCDClient(URL, chainID string, gasPrice msg.DecCoin, gasAdjustment msg.Dec, tmKey key.StdPrivKey) LCDClient {
+	return LCDClient{
+		URL:           URL,
+		ChainID:       chainID,
+		GasPrice:      gasPrice,
+		GasAdjustment: gasAdjustment,
+		TmKey:         tmKey,
+	}
+}
+
+// CreateTxOptions tx creation options
+type CreateTxOptions struct {
+	Msgs []msg.Msg
+	Memo string
+
+	// Optional parameters
+	AccountNumber msg.Int
+	Sequence      msg.Int
+	Fee           tx.StdFee
+}
+
+// CreateAndSignTx build and sign tx
+func (lcdClient LCDClient) CreateAndSignTx(options CreateTxOptions) (tx.StdTx, error) {
```

Comment:
Inconsistent receiver name (use something like `c`), plus you might want to use a pointer receiver to avoid copying the LCDClient object